example/cassiemujoco_ctypes.py

```diff
@@ -551,6 +551,10 @@ class struct_c__SA_drive_filter_t(ctypes.Structure):
 cassie_vis_draw_depth.restype = POINTER_T(ctypes.c_float)
 cassie_vis_draw_depth.argtypes = [POINTER_T(struct_cassie_vis), POINTER_T(struct_cassie_sim), ctypes.c_int32, ctypes.c_int32]
 
+cassie_vis_get_rgb = _libraries['./libcassiemujoco.so'].cassie_vis_get_rgb
+cassie_vis_get_rgb.restype = POINTER_T(ctypes.c_ubyte)
+cassie_vis_get_rgb.argtypes = [POINTER_T(struct_cassie_vis), POINTER_T(struct_cassie_sim), ctypes.c_int32, ctypes.c_int32]
```

Comment:
Nitpick: I think `c_int32` and `c_int` are the same thing, can just change these to `c_int` I think. Feel free to change them in `cassie_vis_draw_depth` too.